From: SVN-Git Migration <python-modules-team@lists.alioth.debian.org>
Date: Thu, 8 Oct 2015 13:39:22 -0700
Subject: build_debug_libs

Build libs and designer plugin with debug for -dbg packages
Patch-Name: build_debug_libs.patch
---
 Qt4Qt5/qscintilla.pro        | 2 +-
 designer-Qt4Qt5/designer.pro | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Qt4Qt5/qscintilla.pro b/Qt4Qt5/qscintilla.pro
index 856854e..f769778 100644
--- a/Qt4Qt5/qscintilla.pro
+++ b/Qt4Qt5/qscintilla.pro
@@ -21,7 +21,7 @@
 !win32:VERSION = 15.0.0
 
 TEMPLATE = lib
-CONFIG += qt warn_off thread exceptions hide_symbols
+CONFIG += qt warn_off thread exceptions debug
 
 CONFIG(debug, debug|release) {
     mac: {
diff --git a/designer-Qt4Qt5/designer.pro b/designer-Qt4Qt5/designer.pro
index f03d3b3..504c14b 100644
--- a/designer-Qt4Qt5/designer.pro
+++ b/designer-Qt4Qt5/designer.pro
@@ -4,7 +4,7 @@
 TEMPLATE = lib
 TARGET = qscintillaplugin
 
-CONFIG += plugin qscintilla2
+CONFIG += plugin qscintilla2 debug
 
 greaterThan(QT_MAJOR_VERSION, 4) {
     QT += designer
